charts/sscs-tribunals-api/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: sscs-tribunals-api
 home: https://github.com/hmcts/sscs-tribunals-case-api
-version: 0.0.94
+version: 0.0.95
```

Comment:
Should this be incremented?

Reply:
un bumped

Reply:
Hasn't been unbumped

Reply:
It cannot be unbumped as it then gets rebumped on github